# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../arcload/99_isofs_blocks.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

mkisofs rounds VH entries to 2048 bytes, so stop parsing on first zero, sigh.

--- arcload-0.5/loader/config.c.orig	2020-07-06 19:19:56.493475375 +0200
+++ arcload-0.5/loader/config.c	2020-07-06 19:24:05.429475496 +0200
@@ -353,6 +353,7 @@
 
 		for(i = 0; i < block; i++) {
 			ch = buf[i];
+			if (!ch) {block = i; break;}
 			switch(t_state) {
 			case T_NONE:
 				if(ch == '#') {
